requirements/dev.txt

```diff
@@ -25,7 +24,7 @@ certifi==2021.10.8
     # via
     #   amlight-coloring
     #   requests
-charset-normalizer==2.0.10
+charset-normalizer==2.0.12
```

Comment:
```suggestion
charset-normalizer==2.0.10
```

@rmotitsuki `2.0.12` would break the Docker build, the common denominator that we current have for the NApps is 2.0.10, could you set this version?